src/features/auth/components/PasswordRule.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FunctionComponent } from 'react'
+
+import { InputRule } from 'ui/components/inputs/InputRule'
+import { Check } from 'ui/svg/icons/Check'
+import { Close } from 'ui/svg/icons/Close'
+import { ColorsEnum } from 'ui/theme'
+
+type Props = {
+  title: string
+  validated: boolean
+}
+
+export const PasswordRule: FunctionComponent<Props> = ({ title, validated }) => {
```

Comment:
J'ai beaucoup de mal avec les variables booléennes qui ne commencent pas par un mot interrogatif. ll faudrait demander l'avis à Victor ;) 
```suggestion
export const PasswordRule: FunctionComponent<Props> = ({ title, isValidated }) => {
```

Reply:
Victor serait d'accord avec toi ahah je modifie